examples/notebooks/persistent_dataset_speed.ipynb

```diff
@@ -0,0 +1,6871 @@
+      "4/16, train_loss: 0.5408, step time: 0.2305\n",
+      "5/16, train_loss: 0.5954, step time: 0.2285\n",
+      "6/16, train_loss: 0.5625, step time: 0.2318\n",
+      "7/16, train_loss: 0.5450, step time: 0.2294\n",
+      "8/16, train_loss: 0.5834, step time: 0.2249\n",
+      "9/16, train_loss: 0.5747, step time: 0.2274\n",
+      "10/16, train_loss: 0.5405, step time: 0.2385\n",
+      "11/16, train_loss: 0.5481, step time: 0.2255\n",
+      "12/16, train_loss: 0.5802, step time: 0.2438\n",
+      "13/16, train_loss: 0.5523, step time: 0.2277\n",
+      "14/16, train_loss: 0.5278, step time: 0.2245\n",
+      "15/16, train_loss: 0.5288, step time: 0.2228\n",
+      "16/16, train_loss: 0.5535, step time: 0.2263\n",
+      "epoch 14 average loss: 0.5558\n",
+      "saved new best metric model\n",
+      "current epoch: 14 current mean dice: 0.0875 best mean dice: 0.0875 at epoch 14\n",
+      "time consuming of epoch 14 is: 9.9445\n",
+      "----------\n",
+      "epoch 15/100\n",
+      "1/16, train_loss: 0.5474, step time: 0.2321\n",
+      "2/16, train_loss: 0.5216, step time: 0.2245\n",
+      "3/16, train_loss: 0.5686, step time: 0.2261\n",
+      "4/16, train_loss: 0.5469, step time: 0.2264\n",
+      "5/16, train_loss: 0.5195, step time: 0.2282\n",
+      "6/16, train_loss: 0.5514, step time: 0.2278\n",
+      "7/16, train_loss: 0.5714, step time: 0.2302\n",
+      "8/16, train_loss: 0.5243, step time: 0.2262\n",
+      "9/16, train_loss: 0.5392, step time: 0.2336\n",
+      "10/16, train_loss: 0.5237, step time: 0.2483\n",
+      "11/16, train_loss: 0.5217, step time: 0.2273\n",
+      "12/16, train_loss: 0.5446, step time: 0.2399\n",
+      "13/16, train_loss: 0.5640, step time: 0.2299\n",
+      "14/16, train_loss: 0.5490, step time: 0.2245\n",
+      "15/16, train_loss: 0.5698, step time: 0.2345\n",
+      "16/16, train_loss: 0.5318, step time: 0.2410\n",
+      "epoch 15 average loss: 0.5434\n",
+      "current epoch: 15 current mean dice: 0.0780 best mean dice: 0.0875 at epoch 14\n",
+      "time consuming of epoch 15 is: 9.3784\n",
+      "----------\n",
+      "epoch 16/100\n",
+      "1/16, train_loss: 0.5472, step time: 0.2297\n",
+      "2/16, train_loss: 0.5629, step time: 0.2252\n",
+      "3/16, train_loss: 0.5016, step time: 0.2242\n",
+      "4/16, train_loss: 0.5295, step time: 0.2268\n",
+      "5/16, train_loss: 0.5475, step time: 0.2247\n",
+      "6/16, train_loss: 0.5102, step time: 0.2268\n",
+      "7/16, train_loss: 0.5621, step time: 0.2585\n",
+      "8/16, train_loss: 0.5338, step time: 0.2284\n",
+      "9/16, train_loss: 0.5329, step time: 0.2256\n",
+      "10/16, train_loss: 0.5357, step time: 0.2282\n",
+      "11/16, train_loss: 0.5770, step time: 0.2380\n",
+      "12/16, train_loss: 0.5167, step time: 0.2237\n",
+      "13/16, train_loss: 0.5344, step time: 0.2264\n",
+      "14/16, train_loss: 0.5219, step time: 0.2273\n",
+      "15/16, train_loss: 0.5284, step ti
```

Comment:
You changed to 100 epochs, but the title is still 600.
Please feel free to update to 100, I am updating all the cache-related documents, examples and notebooks, so I will retrain your notebook, too.
Thanks.

Reply:
Fixed, but I may not be able to re-run the notebook until next weekend to make the image.